UnitySetup/UnitySetup.psm1

```diff
@@ -2713,9 +2713,23 @@ function Update-UnityPackageManagerConfig {
         [int]$SearchDepth = 3,
         [Switch]$VerifyOnly,
         [int]$PATLifetime = 7,
-        [guid]$AzureSubscription
```

Comment:
Interesting - I tried this specifically before suggesting you convert to [guid]. Let me try some things.

Reply:
Confirmed - the issue is that you're assuming that $AzureSubscription is valid and then passing it into `Update-PackageAuthConfig` which ends up throwing the error. I have fixes for that an others about to go up.